reconcile/ocm_clusters.py

```diff
@@ -18,17 +18,50 @@
 QONTRACT_INTEGRATION = "ocm-clusters"
 
 
+def _set_rosa_ocm_attrs(cluster: Mapping[str, Any]):
+    """Cluster account (aws) attribute from App-interface differs from the OCMSpec.
```

Comment:
nit 
```suggestion
    """Cluster account (aws) attribute from app-interface differs from the OCMSpec.
```

and below :)